README.md

```diff
@@ -54,6 +54,8 @@ Creating a chatbot involves 3 steps:
 - add dataset
 - query on the dataset and get answers
 
+You can also specify a custom location for the ChromaDB index by passing a path to the `custom_db_dir` parameter when creating an instance of the EmbedChain class.
```

Comment:
fix this documentation to match the most recent changes in `EmbedChain` construtor